applications/asset_tracker_v2/src/modules/data_module.c

```diff
@@ -79,6 +79,10 @@ static struct cloud_data_neighbor_cells neighbor_cells;
  * have to be buffered.
  */
 static struct cloud_data_modem_static modem_stat;
+/* Size of the static modem (modem_stat) data structure.
+ * Used to provide a size when encoding batch data.
+ */
+#define MODEM_STATIC_SIZE 1
```

Comment:
I think it would be more intuitive it it's called `_COUNT` instead of `_SIZE`, or maybe `_ARRAY_SIZE`

Reply:
Good point. I'll change it to `_COUNT`